proofs/interface.go

```diff
@@ -48,3 +53,26 @@ const (
 	// Filecoin node.
 	LiveMode
 )
+
+// SortedCommRs is a slice of CommRs that has deterministic ordering.
+type SortedCommRs struct {
+	c []CommR
```

Comment:
(Non-blocking) Can we avoid cryptic variable names like this on the Golang side? `c` is not especially clear.